<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileUploader</class>
 <widget class="QWidget" name="FileUploader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logo finden</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="durchsuchen_button">
     <property name="text">
      <string>Datein durchsuchen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="datein_list_widget"/>
   </item>
   <item>
    <widget class="QPushButton" name="hochladen_button">
     <property name="text">
      <string>Hochladen</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
